frontend/src/modules/SimulationTimeSeriesMatrix/view.tsx

```diff
@@ -103,15 +111,19 @@ export const view = ({ moduleContext, workbenchSession, workbenchSettings }: Mod
         : null;
 
     // Create Plot Builder
+    const selectedEnsembleSet = new EnsembleSet(selectedEnsembles);
+    const ensembleDisplayNameGenerator = new EnsembleDisplayNameGenerator(selectedEnsembleSet);
```

Comment:
I do not agree. The goal of the generator is to move the "responsibility" of creating a display name out of the `SubplotBuilder`, see further description above. 

But I'll update the generator to take a list of ensembles, rather than an `EnsembleSet`.